include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t              maxVertexInputBindings;
+    //uint32_t              maxVertexInputAttributeOffset;
+    //uint32_t              maxVertexInputBindingStride;
+    //uint32_t              maxVertexOutputComponents;
+    //uint32_t              maxTessellationGenerationLevel;
+    //uint32_t              maxTessellationPatchSize;
+    //uint32_t              maxTessellationControlPerVertexInputComponents;
+    //uint32_t              maxTessellationControlPerVertexOutputComponents;
+    //uint32_t              maxTessellationControlPerPatchOutputComponents;
+    //uint32_t              maxTessellationControlTotalOutputComponents;
+    //uint32_t              maxTessellationEvaluationInputComponents;
+    //uint32_t              maxTessellationEvaluationOutputComponents;
+    //uint32_t              maxGeometryShaderInvocations;
+    //uint32_t              maxGeometryInputComponents;
+    //uint32_t              maxGeometryOutputComponents;
+    //uint32_t              maxGeometryOutputVertices;
+    //uint32_t              maxGeometryTotalOutputComponents;
+    //uint32_t              maxFragmentInputComponents;
+    //uint32_t              maxFragmentOutputAttachments;
+    //uint32_t              maxFragmentDualSrcAttachments;
+    //uint32_t              maxFragmentCombinedOutputResources;
+    uint32_t maxComputeSharedMemorySize;
+    //uint32_t              maxComputeWorkGroupCount[3];
+    //uint32_t              maxComputeWorkGroupInvocations;
+    uint32_t maxWorkgroupSize[3];
+    //uint32_t              subPixelPrecisionBits;
+    //uint32_t              subTexelPrecisionBits;
+    //uint32_t              mipmapPrecisionBits;
+    //uint32_t              maxDrawIndexedIndexValue;
+    uint32_t maxDrawIndirectCount;
+    //float                 maxSamplerLodBias;
+    float    maxSamplerAnisotropyLog2;
+    uint32_t maxViewports;
+    uint32_t maxViewportDims[2];
+    //float                 viewportBoundsRange[2];
+    //uint32_t              viewportSubPixelBits;
+    //size_t                minMemoryMapAlignment;
+    uint32_t bufferViewAlignment;
+    uint32_t UBOAlignment;
+    uint32_t SSBOAlignment;
+    //int32_t               minTexelOffset;
+    //uint32_t              maxTexelOffset;
+    //int32_t               minTexelGatherOffset;
+    //uint32_t              maxTexelGatherOffset;
+    //float                 minInterpolationOffset;
+    //float                 maxInterpolationOffset;
+    //uint32_t              subPixelInterpolationOffsetBits;
+    //uint32_t              maxFramebufferWidth;
+    //uint32_t              maxFramebufferHeight;
+    //uint32_t              maxFramebufferLayers;
+    //VkSampleCountFlags    framebufferColorSampleCounts;
+    //VkSampleCountFlags    framebufferDepthSampleCounts;
+    //VkSampleCountFlags    framebufferStencilSampleCounts;
+    //VkSampleCountFlags    framebufferNoAttachmentsSampleCounts;
+    //uint32_t              maxColorAttachments;
```

Comment:
This is important, basically the max number of rendertargets the graphics pipeline can write to.
> maxColorAttachments is the maximum number of color attachments that can be used by a subpass in a render pass. The colorAttachmentCount member of the VkSubpassDescription or VkSubpassDescription2 structure must be less than or equal to this limit.

Reply:
`GL_MAX_COLOR_ATTACHMENTS`